### PR TITLE
Document version range support in repository content filtering

### PR DESCRIPTION
**Pull request by ljacomet**

### CODE REVIEW

**Comment by ljacomet** [comment]
@bot-gradle test RFM

**Review by melix — APPROVED** [review]

File: subprojects/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -100,7 +101,27 @@ gradle mAL:cT
 
 Note, that even though the kebab case name matching works with tasks too, the recommendation is still to use camel case for them. 
 
-To learn more about name abbreviation, check out the [user guide](userguide/command_line_interface.html#task_name_abbreviation).  
+To learn more about name abbreviation, check out the [user guide](userguide/command_line_interface.html#task_name_abbreviation).
+
+## Support for version ranges in repository content filtering
+
+With [repository content filtering](userguide/declaring_repositories.html#sec:repository-content-filtering), builds can control which repositories are queried for which dependency.
```
Comment (pioterj) [inline subprojects/docs/src/docs/release/notes.md:108]: How about the following slightly changed version?

Builds can control which repositories are queried for which dependency for better performance and security using [repository content filtering](userguide/declaring_repositories.html#sec:repository-content-filtering).

File: subprojects/docs/src/docs/release/notes.md [hunk 2]
```diff
@@ -100,7 +101,27 @@ gradle mAL:cT
...
+This feature provides performance and security benefits.
+
+With this release, when including or excluding a specific dependency version, the build author can use a version range:
```
Comment (pioterj) [inline subprojects/docs/src/docs/release/notes.md:111]: Is there any use case that we can mention here? I understand what the feature does, I just have no idea why I would do it.

Comment (ljacomet) [reply subprojects/docs/src/docs/release/notes.md:111]: This is mostly going to be used for internal dependencies or allowing / preventing use of some versions following company policies.
But I do not think this is worth describing here.

File: subprojects/docs/src/docs/release/notes.md (hunk 1 above)
Comment (ljacomet) [reply subprojects/docs/src/docs/release/notes.md:108]: Will push the changed version directly.

Comment (ljacomet) [reply subprojects/docs/src/docs/release/notes.md:108]: Done

File: subprojects/docs/src/docs/release/notes.md (hunk 2 above)
Comment (ljacomet) [reply subprojects/docs/src/docs/release/notes.md:111]: Ok with not attempting to add anything there?

**Pull request merged**